### ISSUE TITLE
provisioning does not work in vagrant 2.2.13

### ISSUE THREAD

**Issue by bitboxer** [airyhq/airy#345]
On my machine with the latest vagrant I see this output when running the `bootstrap.sh` script: 

```
    airy-core: Running: inline script
    airy-core: Error from server (Forbidden): pods "startup-helper" is forbidden: error looking up service account default/default: serviceaccount "default" not found
==> airy-core: Trigger run failed
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by ljupcovangelski** [comment]
This looks like a Vagrant issue, as the triggers for `:up` runs after the first provisioner is done (doesn't wait for all three provisioners to finish).

Also another indication for a Vagrant bug is that if we put all three scripts in a single provisioner
```
 airy_core.vm.provision "pre-requisites", type: "shell", env: {"APP_IMAGE_TAG" => ENV['AIRY_VERSION']} do |p|
      p.inline = "
        /vagrant/scripts/provision/prerequisites.sh;
        /vagrant/scripts/provision/core.sh;
        /vagrant/scripts/conf.sh
        "
    end
```
the `start` and `status` triggers run twice.
```
    airy-core: deployment.apps/postgres scaled
==> airy-core: Running action triggers after up ...
==> airy-core: Running trigger: start...
    airy-core: Running: inline script
...
    airy-core: pod "startup-helper" force deleted
==> airy-core: Running trigger: status...
    airy-core: Running: inline script
...
    airy-core: pod "startup-helper" force deleted
==> airy-core: Running action triggers after up ...
==> airy-core: Running trigger: start...
    airy-core: Running: inline script
...
    airy-core: pod "startup-helper" force deleted
==> airy-core: Running trigger: status...
    airy-core: Running: inline script
...
```

Also ssh now doesn't work on Linux
```
$ vagrant ssh
OpenSSL version mismatch. Built against 100020ef, you have 1000106f
$
```

**Comment by bitboxer** [comment]
Can we work around that bug? And have you checked if someone already reported this?

**Comment by pascal-airy** [comment]
Fixed in 0.2.0
